Fixed #32820 -- Added aria-invalid="true" to fields with errors. (#17111)
django/django: docs/ref/forms/api.txt

        >>> str(f["subject"].errors)
        ''

>     When rendering a field with errors, ``aria-invalid="true"`` will be set on 

Reviewer: ```suggestion
    When rendering a field with errors, ``aria-invalid="true"`` will be set on
```

    the field's widget to indicate there is an error to screen reader users.

    .. versionchanged:: 5.0

>         The ``aria-invalid="true"`` is set when a field has errors. 

Reviewer: ```suggestion
        The ``aria-invalid="true"`` was added when a field has errors.
```
